zebra/zebra_vxlan.c

```diff
@@ -1362,7 +1362,7 @@ static int zvni_gw_macip_add(struct interface *ifp, zebra_vni_t *zvni,
 		zlog_debug(
 			"%u:SVI %s(%u) VNI %u, sending GW MAC %s IP %s add to BGP",
 			ifp->vrf_id, ifp->name, ifp->ifindex, zvni->vni,
-			prefix_mac2str(macaddr, NULL, ETHER_ADDR_STRLEN),
+			prefix_mac2str(macaddr, buf, ETHER_ADDR_STRLEN),
```

Comment:
Thi should be consistent with line 1347, i.e. either ETHER_ADDR_STRLEN or  sizeof(buf) in both places.